src/rastervision/ml_tasks/object_detection.py

```diff
@@ -69,6 +69,10 @@ def make_pos_windows(image_extent, label_store, options):
         return _make_label_pos_windows(image_extent, label_store, options)
     elif window_method == 'image':
         return [image_extent.make_copy()]
+    elif window_method == 'sliding':
```

Comment:
Confused about this part - we're making positive windows out of the whole extent? Is this not really making positive windows, but just making windows that might  have no labels? how does that effect the make_neg_windows? How does this work downstream when the label store is queried for labels for these extents?

Reply:
I can see why this is confusing. The `make_positive_windows` function with the `sliding` options is returning windows that are a mix of positive and negative. I think I can can make this clearer by just moving the code for `sliding` up a level to the `get_train_windows` method and not invoke the notions of positive and negative windows .

Reply:
As it stands, I'm just setting the `neg_ratio` to 0 so there are no extra negative windows created. There's nothing special going on with the label store -- it can be queried for arbitrary windows. It's not like the classification label store that can only be queried for grid aligned windows.

Reply:
OK. Maybe there should be an issue to refactor in the future, to simplify/clarify the pos/neg window creation in this case? Might be a good starter issue for someone.

Reply:
I refactored to make this clearer, but I forgot to put the change in a separate commit. It's here https://github.com/azavea/raster-vision/pull/349/commits/e9a30127969495c96042f40efebfff00df4374d5